x-pack/plugins/apm/public/components/shared/transactions_table/index.tsx

```diff
@@ -337,11 +334,21 @@ export function TransactionsTable({
           <OverviewTableContainer
             fixedHeight={fixedHeight}
             isEmptyAndNotInitiated={
-              transactionGroupsTotalItems === 0 && isNotInitiated
+              transactionGroupsTotalItems === 0 &&
+              status === FETCH_STATUS.NOT_INITIATED
             }
           >
             <EuiBasicTable
-              loading={isLoading}
+              loading={status === FETCH_STATUS.LOADING}
+              noItemsMessage={
+                status === FETCH_STATUS.LOADING
```

Comment:
Do we need to take `NOT_INITIATED` into account here (maybe not, just asking)

Reply:
We don't need `NOT_INITIATED` in this case. As soon as we reach the page, after we verified we have transactionType, we start loading the transactions table and enter in `LOADING` state.